src/cascade/input_data/emr.py

```diff
@@ -105,7 +108,7 @@ def _emr_from_sex_and_node_specific_csmr_and_prevalence(csmr, prevalence):
     mean_interp, stderr_interp = _make_interpolators(csmr)
 
     emr = prevalence[["age_lower", "age_upper", "time_lower", "time_upper", "sex_id", "node_id", "density", "weight"]]
-    emr["measure"] = "mtexcess"
+    emr = emr.assign(measure="mtexcess")
```

Comment:
Is this cleaner than using `.copy()` and keeping the slice assignment syntax? Maybe it is...

Reply:
Erm, this was one of the ones that was "assignment to a slice." I reread the Pandas docs a month or two ago, and they said "assign" was the new thing to use. I'm trying to be obedient to norms (where I can).